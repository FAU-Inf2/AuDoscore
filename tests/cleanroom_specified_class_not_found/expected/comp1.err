Exception in thread "main" java.lang.IllegalArgumentException: Error - Class [Foo] specified in @CompareInterface does not exist in cleanroom
	at CheckAnnotation.getCleanroomClass(CheckAnnotation.java:39)
	at CheckAnnotation.checkAnnotations(CheckAnnotation.java:103)
	at CheckAnnotation.main(CheckAnnotation.java:179)
make: *** [compile-stage1] Error 1
