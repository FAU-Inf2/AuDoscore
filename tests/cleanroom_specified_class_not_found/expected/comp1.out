rm -f *.class */*.class
cp UnitTest.java UnitTest.java.orig
javac -cp lib/tools.jar:lib/junit.jar:lib/junitpoints.jar -proc:only -processor tools.bomacon.BonusMalusConverter UnitTest.java > UnitTest.java.tmp
mv UnitTest.java.tmp UnitTest.java
javac -cp lib/tools.jar:lib/junit.jar:lib/junitpoints.jar -proc:only -processor FullQualifier UnitTest.java > UnitTest.java.tmp
mv UnitTest.java.tmp UnitTest.java
make -B UnitTest.class || ( mv UnitTest.java.orig UnitTest.java; /bin/false; )
make[1]: Entering directory `/home/cubu/AuDoscore/tests/cleanroom_specified_class_not_found/test.31603'
javac -cp lib/json-simple-1.1.1.jar:lib/junit.jar:lib/junitpoints.jar:. UnitTest.java ToTest.java 
make[1]: Leaving directory `/home/cubu/AuDoscore/tests/cleanroom_specified_class_not_found/test.31603'
mv UnitTest.java.orig UnitTest.java
javac cleanroom/*
java -cp lib/tools.jar:lib/junit.jar:lib/junitpoints.jar CheckAnnotation UnitTest 
